packages/grid/src/GridColumn.tsx

```diff
@@ -15,7 +15,10 @@ import { GridColumnModel, GridRowModel } from "./Grid";
 
 export type GridColumnPin = "left" | "right" | null;
 
-export interface GridCellProps<T> {
+type CellValidationState = "error" | "warning" | "success" | undefined;
```

Comment:
do we need undefined if optional?

Reply:
Hmm, I son't remember why I did this 😅

Reply:
Ah, yeah, I just tried removing it and typescript was complaining. The issue is, that getValidationState is optional but that does not mean the return value is optional as well. You have to make it explicitly so.

Reply:
would it be better to have getValidationStatus?: (value: GridCellValueProps<T>) => CellValidationState | undefined ?

Reply:
Probably, good idea 👍